egs/multi_ja_en/ASR/prepare.sh

```
@@ -31,55 +33,54 @@
   if [ -e ../../reazonspeech/ASR/data/manifests/.reazonspeech.done ]; then
-    cd data/fbank
-    ln -svf $(realpath ../../../../reazonspeech/ASR/data/manifests/reazonspeech_cuts*) .
-    cd ..
-    mkdir -p manifests
-    cd manifests
-    ln -svf $(realpath ../../../../reazonspeech/ASR/data/manifests/feats_*) .
-    cd ../..
+    cd data/manifests
+    mkdir -p reazonspeech
+    cd reazonspeech
+    ln -svfr $(realpath ../../../../../reazonspeech/ASR/data/manifests/reazonspeech_cuts*) .
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)